CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # bedrock-web-wallet ChangeLog
 
+## 11.2.0 - 2023-TBD
+
+### Added
+- Check if the signer type is included in "acceptedProofTypes" and throw if it
+  isn't in `presentations.sign()`, i.e if "acceptedProofTypes" is specified,
+  then "Ed25519Signature2020" must be included in the list as one of the types.
```

Comment:
So this is an internal implementation detail that I don't think we need in the changelog. I also don't think someone reading the changelog would easily understand what the API implications are without reading the code around this (what is "signer"?). So, we want this item to be more clear about how it impacts someone using the library. Taking an attempt at that:

```suggestion
- Throw an error if the profile used in `presentations.sign()` (by passing `profileId`)
  cannot sign using one of the passed `acceptedProofTypes`. Practically speaking, only
  the `Ed25519Signature2020` proof type is supported at this time.
```